cl/simple_pages/templates/includes/available_fields.json

```diff
@@ -350,5 +350,61 @@
       "parentheticals":false,
       "recap_archive":true,
       "oral_arguments":false
+   },
+  {
+      "field":"jurisdictionType",
+      "description":"Stands for jurisdiction in RECAP XML docket. For example, 'Diversity', 'U.S. Government Defendant'.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"chapter",
+      "description":"In bankruptcy dockets, the chapter the bankruptcy is currently filed under.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"trustee_str",
+      "description":"In bankruptcy dockets, the name of the trustee handling the case.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"entry_number",
+      "description":"# on the PACER docket page. For appellate cases, this may be the internal PACER ID for the document, when an entry ID is otherwise unavailable.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"pacer_doc_id",
+      "description":"The ID of the document in PACER.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"plain_text",
+      "description":"The ID of the document in PACER.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
+   },
+  {
+      "field":"cites",
+      "description":"All of the Opinion IDs that cite a RECAP Document.",
+      "opinions":false,
+      "parentheticals":false,
+      "recap_archive":true,
+      "oral_arguments":false
    }
```

Comment:
This isn't ready yet, so perhaps we should leave it off until it is?

Reply:
Sure, I've hidden this field for now.